src/main/java/io/mosip/mimoto/constant/SwaggerLiteralConstants.java

```diff
@@ -0,0 +1,77 @@
+
+    /*  IDP Controller */
+    public static final String IDP_NAME = "Wallet Binding" ;
+    public static final String IDP_DESCRIPTION = "All the Authorization Related Endpoints" ;
+    public static final String IDP_BINDING_OTP_SUMMARY = "Invoke OTP Request for Wallet Binding" ;
+    public static final String IDP_BINDING_OTP_DESCRIPTION = "This endpoint allow you to invoke OTP for wallet binding" ;
+    public static final String IDP_WALLET_BINDING_SUMMARY = "Wallet Binding" ;
+    public static final String IDP_WALLET_BINDING_DESCRIPTION = "This endpoint allow you to perform the wallet binding" ;
+    public static final String IDP_GET_TOKEN_SUMMARY = "Retrieve AccessToken for OIDC Flow" ;
+    public static final String IDP_GET_TOKEN_DESCRIPTION = "This endpoint allow you to retrieve the access token in exchange for authorization Code" ;
+
+    /*  Issuers Controller */
+    public static final String ISSUERS_NAME = "Issuers" ;
+    public static final String ISSUERS_DESCRIPTION = "All the Issuers Related Endpoints" ;
+    public static final String ISSUERS_GET_ISSUERS_SUMMARY = "Retrieve All Onboarded Issuers" ;
+    public static final String ISSUERS_GET_ISSUERS_DESCRIPTION = "This endpoint allow you to retrieve all the onboarded issuers" ;
+    public static final String ISSUERS_GET_SPECIFIC_ISSUER_SUMMARY = "Retrieve Specific Issuer's Config" ;
+    public static final String ISSUERS_GET_SPECIFIC_ISSUER_DESCRIPTION = "This endpoint allow you to retrieve the Complete configuration of the specific issuer" ;
+    public static final String ISSUERS_GET_ISSUER_WELLKNOWN_SUMMARY = "Retrieve Specific Issuer's Well known" ;
+    public static final String ISSUERS_GET_ISSUER_WELLKNOWN_DESCRIPTION = "This endpoint allow you to retrieve the well known of the specific issuer" ;
+
+    /*  Prensentation Controller */
+    public static final String PRESENTATION_NAME = "Presentation" ;
+    public static final String PRESENTATION_DESCRIPTION = "All the Online Sharing Related Endpoints" ;
+    public static final String PRESENTATION_AUTHORIZE_SUMMARY = "Perform the Authorization" ;
+    public static final String PRESENTATION_AUTHORIZE_DESCRIPTION = "This endpoint allow you to redirect the token back to the caller post authorization" ;
+
+    /*  Resident Service Controller */
+    public static final String RESIDENT_NAME = "Resident Service" ;
+    public static final String RESIDENT_DESCRIPTION = "All the Resident Service Related Endpoints" ;
+    public static final String RESIDENT_REQUEST_OTP_SUMMARY = "Request for OTP" ;
+    public static final String RESIDENT_REQUEST_OTP_DESCRIPTION = "This endpoint allow you to request OTP for credential Download" ;
+    public static final String RESIDENT_REQUEST_INDIVIDUALID_OTP_SUMMARY = "Request OTP for retrieving individual Id" ;
+    public static final String RESIDENT_REQUEST_INDIVIDUALID_OTP_DESCRIPTION = "This endpoint allow you to request OTP to download individual ID" ;
```

Comment:
Replace ID by Id and individual by Individual to keep it consisent

Reply:
Updated